### PR TITLE
docs(README): Include use cases in introduction

### PR DESCRIPTION
**Pull request by tsteenbe**
Outline in a few bullet points what a user can do with ORT.

Based on materials I have used at conferences and meetups to explain what ORT does in a nutshell and how its different from other solutions on the market.

This is the first pull request of a series of changes to make ORT documentation more complete and easier to understand. In follow up commits some of the words will be turned into links to new sections of the documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,12 +31,16 @@
 
 # Introduction
 
-The OSS Review Toolkit (ORT) aims to assist with the tasks that commonly need to be performed in the context of license
-compliance checks, especially for (but not limited to) Free and Open Source Software dependencies.
+The OSS Review Toolkit (ORT) is a FOSS policy automation and orchestration toolkit which you can use to manage your (open source) software dependencies in a strategic, safe and efficient manner.
```
Comment (mnonnenmacher) [inline README.md:34]: Commit message nits:
* "a user"
* double space after "few"

File: README.md [hunk 2]
```diff
@@ -31,12 +31,16 @@
...
 
-It does so by orchestrating a _highly customizable_ pipeline of tools that _abstract away_ the underlying services.
-These tools are implemented as libraries (for programmatic use) and exposed via a command line interface (for scripted
-use):
+You can use it to:
+
+- Generate CycloneDX, SPDX SBOMs or custom FOSS attribution docs for your software project
+- Automate your FOSS policy using risk-based Policy as Code to do licensing, security vulnerability checks, InnerSource and engineering standards checks for your software project and its dependencies
+- Create a source code archive for your software project, including its dependencies to comply with certain licenses or have your own copy as nothing on the internet is forever
+- Correct package metadata or licensing findings yourself, using InnerSource or with the help of the FOSS community
```
Comment (mnonnenmacher) [inline README.md:41]: All sentences are missing dots at the end, and please limit line length to 120 chars.

File: README.md [hunk 3]
```diff
@@ -31,12 +31,16 @@
...
-The OSS Review Toolkit (ORT) aims to assist with the tasks that commonly need to be performed in the context of license
-compliance checks, especially for (but not limited to) Free and Open Source Software dependencies.
+The OSS Review Toolkit (ORT) is a FOSS policy automation and orchestration toolkit which you can use to manage your (open source) software dependencies in a strategic, safe and efficient manner.
 
-It does so by orchestrating a _highly customizable_ pipeline of tools that _abstract away_ the underlying services.
-These tools are implemented as libraries (for programmatic use) and exposed via a command line interface (for scripted
-use):
+You can use it to:
+
+- Generate CycloneDX, SPDX SBOMs or custom FOSS attribution docs for your software project
```
Comment (mnonnenmacher) [inline README.md:38]: Maybe: "Generate CycloneDX or SPDX BOMs, or custom ..."

File: README.md [hunk 4]
```diff
@@ -31,12 +31,16 @@
...
+- Automate your FOSS policy using risk-based Policy as Code to do licensing, security vulnerability checks, InnerSource and engineering standards checks for your software project and its dependencies
+- Create a source code archive for your software project, including its dependencies to comply with certain licenses or have your own copy as nothing on the internet is forever
+- Correct package metadata or licensing findings yourself, using InnerSource or with the help of the FOSS community 
+
+ORT can be used as library (for programmatic use), via a command line interface (for scripted use), or via its CI integrations and consists out of the following parts:
```
Comment (mnonnenmacher) [inline README.md:43]: * "consists of"
* "tools" instad of "parts"?

File: README.md [hunk 5]
```diff
@@ -31,12 +31,16 @@
...
+The OSS Review Toolkit (ORT) is a FOSS policy automation and orchestration toolkit which you can use to manage your (open source) software dependencies in a strategic, safe and efficient manner.
 
-It does so by orchestrating a _highly customizable_ pipeline of tools that _abstract away_ the underlying services.
-These tools are implemented as libraries (for programmatic use) and exposed via a command line interface (for scripted
-use):
+You can use it to:
+
+- Generate CycloneDX, SPDX SBOMs or custom FOSS attribution docs for your software project
+- Automate your FOSS policy using risk-based Policy as Code to do licensing, security vulnerability checks, InnerSource and engineering standards checks for your software project and its dependencies
+- Create a source code archive for your software project, including its dependencies to comply with certain licenses or have your own copy as nothing on the internet is forever
```
Comment (mnonnenmacher) [inline README.md:40]: Maybe: "software project and its dependencies"

**Review by mnonnenmacher — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (tsteenbe) [reply README.md:38]: Grammatically the 'or' is redundant - its a,b, or c.

File: README.md [hunk 6]
```diff
@@ -31,12 +31,20 @@
 
 # Introduction
 
-The OSS Review Toolkit (ORT) aims to assist with the tasks that commonly need to be performed in the context of license
-compliance checks, especially for (but not limited to) Free and Open Source Software dependencies.
+The OSS Review Toolkit (ORT) is a FOSS policy automation and orchestration toolkit which you can use to manage your
+(open source) software dependencies in a strategic, safe and efficient manner.
```
Comment (fviernau) [inline README.md:35]: should we append: "...on an organization-wide scale" to the end of line 35?

File: README.md [hunk 7]
```diff
@@ -31,12 +31,20 @@
...
 
-It does so by orchestrating a _highly customizable_ pipeline of tools that _abstract away_ the underlying services.
-These tools are implemented as libraries (for programmatic use) and exposed via a command line interface (for scripted
-use):
+You can use it to:
+
+- Generate CycloneDX, SPDX SBOMs, or custom FOSS attribution docs for your software project
```
Comment (fviernau) [inline README.md:39]: maybe say "documents" instead of just "docs".

File: README.md [hunk 8]
```diff
@@ -31,12 +31,20 @@
...
-The OSS Review Toolkit (ORT) aims to assist with the tasks that commonly need to be performed in the context of license
-compliance checks, especially for (but not limited to) Free and Open Source Software dependencies.
+The OSS Review Toolkit (ORT) is a FOSS policy automation and orchestration toolkit which you can use to manage your
+(open source) software dependencies in a strategic, safe and efficient manner.
 
-It does so by orchestrating a _highly customizable_ pipeline of tools that _abstract away_ the underlying services.
-These tools are implemented as libraries (for programmatic use) and exposed via a command line interface (for scripted
-use):
+You can use it to:
+
```
Comment (fviernau) [inline README.md:38]: I believe NOTICE files would be worth mentioning in the list of attribution documents below.

File: README.md [hunk 9]
```diff
@@ -31,12 +31,20 @@
...
+- Generate CycloneDX, SPDX SBOMs, or custom FOSS attribution docs for your software project
+- Automate your FOSS policy using risk-based Policy as Code to do licensing, security vulnerability checks, InnerSource
+and engineering standards checks for your software project and its dependencies
```
Comment (fviernau) [inline README.md:41]: I believe the first "checks", after "vulnerability" should be removed.

File: README.md [hunk 10]
```diff
@@ -31,12 +31,20 @@
...
+- Create a source code archive for your software project and its dependencies to comply with certain licenses or have
```
Comment (fviernau) [inline README.md:42]: nit: Regarding "software project and its dependencies" would it make sense to assume the audience has mutliple software projects, e.g. say "software projects and their dependencies"?

File: README.md [hunk 11]
```diff
@@ -31,12 +31,20 @@
...
+your own copy as nothing on the internet is forever
```
Comment (fviernau) [inline README.md:43]: consider "as nothing on the internet is forever" -> "as a backup"

**Review by fviernau — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (tsteenbe) [reply README.md:35]: No, like to keep it short & simple - rest of the doc update will make it clear ORT is great for orgs.

File: README.md (hunk 8 above)
Comment (tsteenbe) [reply README.md:38]: FOSS attribution docs = NOTICE files - bullet points are no meant to describe every feature but as short & simple marketing messaging - in follow up commit I plan to add links to new sections that add the details.

File: README.md (hunk 11 above)
Comment (tsteenbe) [reply README.md:43]: Marketing copy "as nothing on the internet is forever" is stronger and easy to remember statement. The bullet point is hinting at business continuity risk which you don't get across using "as a backup"

File: README.md (hunk 10 above)
Comment (tsteenbe) [reply README.md:42]: We made it singular as:
- Checks that ORT supports are usually done on a project-by-project basis
- ORT out of the box does not have a feature to scan all software projects an person or org may have - say scanning an entire GitHub or GitLab org.

**Review by fviernau — APPROVED** [review]

**Review by fviernau — APPROVED** [review]

**Pull request merged**